### PR TITLE
Distribute check

### PR DESCRIPTION
**Pull request by kamikazebr**

### CODE REVIEW

File: pkg/contracts/src/CVStrategy.sol [hunk 1]
```diff
@@ -167,13 +168,11 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
...
     uint256 public constant D = 10000000; //10**7
-    uint256 public constant PRECISION_PERCENTAGE = 100 * PRECISION_SCALE;
-    // uint256 public constant ONE_HUNDRED_PERCENT = 1e18;
+    // uint256 public constant PRECISION_PERCENTAGE = 100 * PRECISION_SCALE;
     uint256 private constant TWO_128 = 0x100000000000000000000000000000000; // 2**128
     uint256 private constant TWO_127 = 0x80000000000000000000000000000000; // 2**127
     uint256 private constant TWO_64 = 0x10000000000000000; // 2**64
-    //    uint256 public constant ABSTAIN_PROPOSAL_ID = 1;
-    uint256 public constant MAX_STAKED_PROPOSALS = 10; //@todo not allow stake more than 10 proposals per user
+    uint256 public constant MAX_STAKED_PROPOSALS = 10; //@todo not allow stake more than 10 proposals per user, dont count executed?
```
Comment (rperez89) [inline pkg/contracts/src/CVStrategy.sol:175]: Have you ever think about increasing it? or is too havy in terms of costs for the searchs with the for(....)

File: pkg/contracts/src/CVStrategy.sol [hunk 2]
```diff
@@ -742,7 +690,7 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
             proposal.voterStakedPointsPct[_sender] = stakedPointsPct;
 
             // console.log("_sender", _sender);
-            uint256 stakedAmount = convertPctToTokens(stakedPointsPct);
+            uint256 stakedAmount = stakedPointsPct;
```
Comment (rperez89) [inline pkg/contracts/src/CVStrategy.sol:693]: could we also get rid of that stakedPointsPct?

File: pkg/subgraph/subgraph.yaml [hunk 3]
```diff
@@ -6,15 +6,15 @@ schema:
 dataSources:
   - kind: ethereum/contract
     name: RegistryFactory
-    network: localhost
+    network: arbitrum-sepolia
```
Comment (rperez89) [inline pkg/subgraph/subgraph.yaml:9]: those changes are not going to affect the localhost env?

File: pkg/contracts/src/CVStrategy.sol (hunk 1 above)
Comment (kamikazebr) [reply pkg/contracts/src/CVStrategy.sol:175]: we dont have it implented that yet, so idk how much that will cost already for 10. also that impact the UI in some way.

File: pkg/contracts/src/CVStrategy.sol (hunk 2 above)
Comment (kamikazebr) [reply pkg/contracts/src/CVStrategy.sol:693]: yeah i want give big refactor after we get everything merged. i can test doing just that change names for that branch at least.

File: pkg/subgraph/subgraph.yaml (hunk 3 above)
Comment (kamikazebr) [reply pkg/subgraph/subgraph.yaml:9]: should affect but because people run local command that its overriden.

**Comment by rperez89** [comment]
Got it !

File: pkg/contracts/src/CVStrategy.sol (hunk 2 above)
Comment (kamikazebr) [reply pkg/contracts/src/CVStrategy.sol]: renamed

**Pull request merged**